docs/operation/listening_processes.rst

```diff
@@ -0,0 +1,29 @@
+Listening processes
```

Comment:
Title-case in our docs is requiring us to always capitalize words (unless they are for liaison): `Listening Processes` in this case.